firmware-support/memmap-generate/src/generators/mod.rs

```diff
@@ -31,10 +31,10 @@ pub enum IdentType {
 /// Generate a contextual identifier from a string.
 pub fn ident(ident_type: IdentType, n: impl AsRef<str>) -> Ident {
     let s = match n.as_ref() {
-        "(,)" => "Pair",
-        "(,,)" => "Triple",
-        "(,,,)" => "FourTuple",
-        "(,,,,)" => "FiveTuple",
+        "Tuple2" => "Pair",
+        "Tuple3" => "Triple",
+        "Tuple4" => "FourTuple",
+        "Tuple5" => "FiveTuple",
```

Comment:
@hydrolarus Should we just remove this now?

Reply:
I think the names are fine as they are, so I don't think we'd need this change. If we're sure that we will never run this code on memory maps created with a GHC < 9.6 then we could remove it, otherwise maybe I'd keep it for compat and change the names to `Tuple2` etc to be consistent with newer versions?